doc/source/ray-client.rst

```diff
@@ -0,0 +1,65 @@
+**********
+Ray Client
+**********
+
+.. note::
+
+   This feature is still in beta and subject to changes.
+
+=====
+Basic usage
+=====
+
+While in beta, the server is available as an executable module. To start the server, run
+
+``python -m ray.util.client.server [--host host_ip] [--port port]``
+
+This runs ``ray.init()`` with default options and exposes the client gRPC port at ``host_ip:port`` (by default, ``0.0.0.0:50051``)
+
+From here, another Ray script can access that server from a networked machine with ``ray.connect()``
+
+.. code-block:: python
```

Comment:
I think there's something wrong with the indentation here, the code blocks doesn't show up when I compile the docs locally.